modin/pandas/series.py

```diff
@@ -543,7 +543,7 @@ def values(self):  # noqa: RT01, D200
         """
         Return Series as ndarray or ndarray-like depending on the dtype.
         """
```

Comment:
was considering putting a:
```
UserWarning: 'Series.values` defaulting to to_numpy().
```

Reply:
This is not done for DataFrame.values.

Having this message for both DataFrame.values and Series.values is probably not super useful, except in the case where we're operating on an empty DataFrame/Series and we're already surfacing a 
```
UserWarning: 'DataFrame.to_numpy` for empty DataFrame defaulting to pandas implementation.
```
in which case it will be confusing why DataFrame/Series using to_numpy and defaulting to pandas when the user is calling .values? (and the "values defaulting to to_numpy()" message would clarify).